tag_processor_plugin/tag_processor_plugin.go

```
@@ -0,0 +1,582 @@
+
+	return resultBatch, nil
+}
+
+// validateMessage checks if a message has all required metadata fields
+func (p *TagProcessor) validateMessage(msg *service.Message) error {
+	requiredFields := []string{"location_path", "data_contract", "tag_name"}
+	missingFields := []string{}
+
+	// Get current metadata for context
+	metadata := map[string]string{}
+	msg.MetaWalkMut(func(key string, value any) error {
```

golangci-lint (1.62.2)

Error return value of `msg.MetaWalkMut` is not checked

(errcheck)